actioncable/lib/action_cable/channel/base.rb

```diff
@@ -159,10 +159,12 @@ def initialize(connection, identifier, params = {})
       def perform_action(data)
         action = extract_action(data)
 
-        if processable_action?(action)
-          dispatch_action(action, data)
-        else
-          logger.error "Unable to process #{action_signature(action, data)}"
+        ActiveSupport::Notifications.instrument("perform_action.action_cable", action: action, data: data) do
```

Comment:
This should go inside the conditional, right above the dispatch method. Wouldn't we only want to send out notifications for actions that can be processed?